site/docs/get-started/OneClickDemo.md

```diff
@@ -32,12 +34,12 @@ Alternatively you can try Fybrik with Katalog,  a data catalog stub, strictly us
 
 === "Demo with OpenMetaData" 
     ```bash
-    curl https://raw.githubusercontent.com/fybrik/fybrik/master/samples/OneClickDemo/OneClickDemo-OMD.sh | bash -
+    export FYBRIK_VERSION={{ currentRelease|default('master') }}; curl https://raw.githubusercontent.com/fybrik/fybrik/master/samples/OneClickDemo/OneClickDemo-OMD.sh | bash -
```

Comment:
you added the FYBRIK_VERSION env variable, but do not use it.
should it be:
```
curl https://raw.githubusercontent.com/fybrik/fybrik/${FYBRIK_VERSION}/samples/OneClickDemo/OneClickDemo-OMD.sh 
```

Reply:
that was not my intention, the variable is used in the script itself,
but your suggestion is good also, added